orchagent/vxlanorch.cpp

```diff
@@ -1058,6 +1062,43 @@ void VxlanTunnel::updateDipTunnelRefCnt(bool inc, tunnel_refcnt_t& tnl_refcnts,
     }
 }
 
+void VxlanTunnel::updateRemoteEndPointIpRef(const std::string remote_vtep, bool inc)
```

Comment:
The IMR reference count is incremented in createDIP tunnel flow. However in case of P2MP the reference for the remote can be inferred on the bridge ports added to L2MC group member. So it is not separately maintained. Please check isTunnelReferenced
This not only maintains the remote end point ip but also remote VNI too.
        /* Bridge port will have reference since on IMET routes reception L2MC group member
           would be created with end point IP and the P2MP tunnel bridge port */

        if (gPortsOrch->getBridgePortReferenceCount(tunnelPort) != 0)

Reply:
Routes flow come through route orch and thus need to be maintained. So I am reusing existing route reference while considering bridge port references for remote IMR, VNI